queue_services/entity-emailer/src/entity_emailer/email_templates/RES-COMPLETED.jinja2

```diff
@@ -0,0 +1,15 @@
+{% extends 'RES-common.jinja2' %}
+
+{% block page_title %}You have successfully restored your business with the BC Business Registry.{% endblock %}
+
+{% block title_message %}You have successfully restored your business with the BC Business Registry.{% endblock %}
+
+{% block attachments %}
+<ul>
+    <li>Notice of Articles</li>
+    <li>Incorporation Certificate</li>
+</ul>
+{% endblock %}
```

Comment:
The `RES-COMPLETED` template extends the `RES-common` template and replaces the blocks listed above.